src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -200,6 +201,59 @@ class CosmasGoogleCloudServiceTest {
         assertNotNull(stream.error)
     }
 
+    @Test
+    fun simpleAddPatch() {
+        createVersion("file", "1", "1")
+        addPatchToService("abc", "-", "1", 1, "1")
+        commit("1")
```

Comment:
don't you want to check something in the test? How do you know that implementations of these methods are not empty?

Reply:
change tests